src/main/java/school/lemon/changerequest/java/introduction/hw2/Task9.java

```diff
@@ -1,7 +1,45 @@
 package school.lemon.changerequest.java.introduction.hw2;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 
 public class Task9 {
     public static String[] extractData(String URL) {
-        return new String[]{"", "", ""};
+
+        String result[] = new String[3];
```

Comment:
Your solution is really very complicated.

It's possible to extract all data with only one regexp pattern.

Try to use [regex101](https://regex101.com/) to visualize your patterns.